src/protocols/bdx/BdxMessages.cpp

```diff
@@ -246,6 +274,15 @@ size_t SendAccept::MessageSize() const
     return WriteToBuffer(emptyBuf).Needed();
 }
 
+#if CHIP_AUTOMATION_LOGGING
+void SendAccept::LogMessage(bdx::MessageType messageType) const
+{
+    ChipLogAutomation("SendAccept");
```

Comment:
Add `(void)messageType;` in all methods where the argument is ignored to avoid build warnings